presto-main/src/main/java/com/facebook/presto/server/testing/TestingPrestoServer.java

```diff
@@ -861,4 +865,11 @@ private static int driftServerPort(DriftServer server)
     {
         return ((DriftNettyServerTransport) server.getServerTransport()).getPort();
     }
+
+    public static ClientRequestFilterManager getClientRequestFilterManager(List<ClientRequestFilterFactory> requestFilterFactory)
+    {
+        ClientRequestFilterManager manager = new ClientRequestFilterManager();
```

Comment:
Shouldn't this be retrieved from Guice, since you're already creating one in `ClientRequestServerModule`?